x-pack/test/security_solution_endpoint_api_int/apis/fixtures/package_registry_config.yml

```diff
@@ -1,2 +1,4 @@
 package_paths:
   - /packages/production
+  - /packages/staging
```

Comment:
The package is currently in the staging environment which is why we need this. I added the snapshot just in case we ever need to grab one from there.